_includes/head.html

```diff
@@ -7,9 +7,20 @@
 ================================================== -->
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  {% unless site.env == "production" %}
-    <meta name="robots" content="noindex, nofollow">
-  {% endunless %}
+
+  {% if page.expiration_date %}
+    {% assign today_date = 'now' | date: '%s' %}
```

Comment:
Wanted to share for reference, DigitalGov automatically marks any content older than 5 years.

https://github.com/GSA/digitalgov.gov/blob/41168326c783478d4e0c037f0db14508dd601971/themes/digital.gov/layouts/partials/core/blog-date-warning.html#L1-L2

Reply:
Thanks for sharing! I _think_ we are leaning away from automating this just to maintain better control and because we don't update this section too frequently, but that is just my understanding. @sarah-sch and @finekatie, would you prefer if these pages were removed from search after x number of years instead?

Reply:
Right, this suggestion is for future reference.